de.sebastianruziczka.looksyk.yml

```diff
@@ -0,0 +1,56 @@
+app-id: de.sebastianruziczka.looksyk
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+# node24 not yet available for 25.08
+base-version: '25.08'
+command: looksyk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node24
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/node24/bin:/usr/lib/sdk/rust-stable/bin
+  env:
+    CARGO_HOME: cargo-cache
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=home
+  # neede for audio or video files
+  - --socket=pulseaudio
+  - --device=dri
+  - --env=ELECTRON_TRASH=gio
+  - --env=ELECTRON_OZONE_PLATFORM_HINT=auto
+modules:
+  - name: looksyk-desktop
+    buildsystem: simple
+    build-commands:
+      - cd git && make && cd ..
+      # This does not work. flatpak-node-generator does not support git-sources-lockfile-v2 and electron-forge does not
+      # support caching of electron and is trying to fetch it every time with electron/@get, even if caching and offline
+      # mode is active
+      #- cd application-wrapper/Looksyk && npm install --offline --cache ./npm-cache && npm run package --offline --cache ./npm-cache && cd ..
+      - cd frontend/looksyk && npm install --offline --cache ./npm-cache && npm run build --configuration=production && cd ..
+      - cd backend && cargo build --release && cd ..
+      - install -Dm755 backend/target/release/looksyk /app/bin/looksyk-backend
+      - install -Dm755 git/git /app/bin/git
```

Comment:
what's happening here? do you have git source inside your source archive or precompiled git?

Reply:
ah I just saw the git source, I'd separate these two modules and use the git tarball. Why do you need git at runtime?

Reply:
Looksyk uses Git to enable version control and synchronization between devices.
This has been an optional feature since 1.14.0.

https://sebastianrzk.github.io/Looksyk/sync/

I replaced the git-source with a tarball